internal/util/update/resource-merge.go

```diff
@@ -125,7 +126,7 @@ func (u ResourceMergeUpdater) updatePackage(subPkgPath, localPath, updatedPath,
 	// Package deleted from upstream
 	case originalExists && localExists && !updatedExists:
 		// Check the diff. If there are local changes, we keep the subpackage.
-		diff, err := copyutil.Diff(originalPath, localPath)
+		diff, err := pkgdiff.PkgDiff(originalPath, localPath)
```

Comment:
Why we change this?

Reply:
we want to have a unified way to compare diff. We use pkgdiff in rest of the places and it handles the annotation comparison as well.